BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -178,14 +177,15 @@ extension StoresTableViewController {
             
             guard let noResultsCell = tableView.dequeueReusableCell(withIdentifier: "noResultsCell") as? NoResultsCustomCell else {
                 
+                makeNoteThatCellFailedToDequeue(cell: "NoResultsCustomCell")
                 return UITableViewCell()
                 
             }
             
-            checkUserEmailSetting(in: noResultsCell)
-            noResultsCell.onEmailButtonPressed = { noResultsCell in
+            checkIfUserHasAnEmail(whenIn: noResultsCell)
```

Comment:
Again the method name should be more like:
`updateCellWithUserEmailSettings()`, because it updates the cell depending on whether the user has emails or not.
It does more than just check. Remember that chapter about function names in "Clean Code"?